authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java

```diff
@@ -51,17 +51,17 @@ protected String extractBioData(String dataFieldValue) throws IdAuthenticationAp
 
 	@Override
 	protected boolean isSigningRequired() {
-		return env.getProperty("mosip.ida.internal.signing-required", Boolean.class, true);
+		return false;

```

Comment:
This should return `true`.

Reply:
@LoganathanSekar7627 as per story "For new internal API, signature validation is enabled and all other validations are disabled."

Reply:
@manojsp12 , this property is for the output signature response header,  which always should be there for all responses. This should be true.

Reply:
@LoganathanSekar7627 Updated